plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/AbstractBuildBootableJarMojo.java

```diff
@@ -301,24 +331,31 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         } catch (ProvisioningException | IOException | XMLStreamException ex) {
             throw new MojoExecutionException("Provisioning failed", ex);
         }
+
         try {
+            // Legacy Patching point
+            legacyPatching();
             copyExtraContentInternal(wildflyDir, contentDir);
             copyExtraContent(wildflyDir);
             List<String> commands = new ArrayList<>();
             deploy(commands);
-            configureCli(commands);
+            List<String> serverConfigCommands = new ArrayList<>();
+            configureCli(serverConfigCommands);
+            commands.addAll(serverConfigCommands);
             if (!commands.isEmpty()) {
-                executeCliScript(wildflyDir, commands, null, false, "Server configuration");
-                // Store generated commands to file in build artifacts.
-                Path genCliScript = contentRoot.resolve("generated-cli-script.txt");
-                try (FileWriter writer = new FileWriter(genCliScript.toFile())) {
-                    for(String str: commands) {
-                        writer.write(str + System.lineSeparator());
+                executeCliScript(wildflyDir, commands, null, false, "Server configuration", true);
+                if (!serverConfigCommands.isEmpty()) {
+                    // Store generated commands to file in build artifacts.
+                    Path genCliScript = contentRoot.resolve("generated-cli-script.txt");
+                    try (FileWriter writer = new FileWriter(genCliScript.toFile())) {
```

Comment:
This shouldn't block merging but we _may_ want to use:
```
try (BufferedWriter writer = FIles.newBufferedWriter(genCliSript, StandardCharsets.UTF-8)) {
    for (String str : serverConfigCommans) {
        writer.write(str);
        writer.newLine():
    }
}
````

Really just the encoding is what we may want.

Reply:
Thank-you, fixed.